package.json

```diff
@@ -67,45 +67,46 @@
     "babel-preset-stage-1": "^6.0.0",
     "babel-standalone": "^6.4.4",
     "codemirror": "^5.15.2",
-    "css-loader": "~0.9.0",
     "prop-types": "^15.5.10",
     "react": "^15.4.2",
     "react-addons-test-utils": "^15.4.2",
     "react-codemirror": "^1.0.0",
     "react-dom": "^15.4.2",
     "rimraf": "^2.4.0",
-    "style-loader": "~0.8.0",
-    "url-loader": "~0.5.5",
-    "webpack": "^1.10.0"
+    "webpack": "^1.12.3"
```

Comment:
Can we ticket out upgrading to `webpack@3` later?

Reply:
Done: https://github.com/FormidableLabs/component-playground/issues/102